package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "hapi-fxa-oauth",
+  "description": "A simple authentication plugin for Hapi applications to become Firefox Accounts OAuth service providers",
   "version": "2.0.1",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "MPL 2.0",
+  "author": "Mozilla (https://mozilla.org/)",
   "dependencies": {
     "boom": "2.6.1",
     "poolee": "0.4.15"
   },
+  "devDependencies": {
+    "eslint": "^1.7.3",
+    "eslint-config-fxa": "^2.0.0"
+  },
+  "license": "MPL-2.0",
+  "main": "index.js",
   "peerDependencies": {
     "hapi": ">=8.x.x"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "npm run lint"
```

Comment:
I support this change but I think we should put the "Error: no test specified" back in after it, to keep us honest.  I'll do that in a follow-up PR.